packages/mui-material/src/styles/CssVarsProvider.tsx

```diff
@@ -1,9 +1,9 @@
-'use client';
 // do not remove the following import (https://github.com/microsoft/TypeScript/issues/29808#issuecomment-1320713018)
 /* eslint-disable @typescript-eslint/no-unused-vars */
 // @ts-ignore
 import * as React from 'react';
```

Comment:
Dead logic?

```suggestion
```

Reply:
Required, see https://ci.codesandbox.io/status/mui/material-ui/pr/41131/builds/473769.

Reply:
Ah right, @Janpot left a comment about this above. The .d.ts file generated https://unpkg.com/browse/@mui/material@5.15.10/styles/CssVarsProvider.d.ts needs this import.

Reply:
Would the following work as well without the `@ts-ignore`?
```tsx
/// <reference types="react" />
```